web/src/constants/SpanAttribute.constants.ts

```diff
@@ -8,10 +8,14 @@ export const TraceTestAttributes = {
   TRACETEST_RESPONSE_STATUS: 'tracetest.response.status',
   TRACETEST_RESPONSE_BODY: 'tracetest.response.body',
   TRACETEST_RESPONSE_HEADERS: 'tracetest.response.headers',
-  TRACETEST_SELECTED_SPANS_COUNT: 'tracetest.selected_spans.count'
+  TRACETEST_SELECTED_SPANS_COUNT: 'tracetest.selected_spans.count',
 };
 
-export const Attributes = {
+export const AttributesTags: Record<string, string[]> = {
+  TRACETEST_SPAN_DURATION: ['ms', 'time'],
+};
```

Comment:
Looks good man. We need to create more tags and descriptions for attributes following the OTel semantic convention https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions

Reply:
Yes, the idea is to grab each of description text and add show it as part of the attribute input in the create assertion form.

Reply:
OK. Will add those I find on the docs. But will leave a placeholder for all the one we go so far.

Reply:
Can we leverage the yaml files that contain these definitions, for example: https://github.com/open-telemetry/opentelemetry-specification/blob/main/semantic_conventions/trace/http.yaml